README.md

```diff
@@ -1,43 +1,77 @@
+> We prefer a containerized development. Therefore, the only dependeciy needed to work on this project is [docker/ docker-compose](https://docs.docker.com/get-docker/). To make this easier for new developers we have created a script that can be  used to run the api in a container locally, this script is placed in the project root and is called ```dev_utils.sh```.
 
+> We currently do not have a development environment for our backend, so when working on our frontend you should pull our [backend repo](https://github.com/td-org-uit-no/tdctl-api) and run it locally. The frontend container will then automatically try to connect to your local backend.
+
+1. Clone the project
+
+```bash
+    git clone git@github.com:td-org-uit-no/tdctl-frontend.git
 ```
-yarn start:local
+2. Go to the project directory
+
+```bash
+    cd tdctl-frontend
 ```
 
-this will use a different set of enivronment variables for defining the backend URL. You can see these over in the `./environments` folder.
+3. Add executable rights to the container utils script
+```bash
+    chmod +x ./dev_utils
+```
 
-## :clap: Contributing
+4. Build docker container
+> This is only needed if it is the first time your running the project or if there has been any changes to the runtime environment of the website. 
+```bash
+    ./dev_utils compose build
+```
 
-### 💡 Got an idea for a feature, or got a bug to report?
+1. Launch the container
+    * Run container in background
+        > Shutdown the container by running ```./dev_utils compose down```
+        ```bash
+            ./dev_utils compose up -d
+        ```
+        - You can now start a interactive shell within the container by running the command
+            > To get an interactive shell in the database container add ```db``` to the end of this command
+            ```bash
+                ./dev_utils exec
+            ```
+    * Run the container and view container output
+        > Shutdown the container by pressing ```ctrl-C```
+        ```bash
+            ./dev_utils compose up
+        ```
 
-[Create an issue](https://github.com/td-org-uit-no/tdctl-frontend/issues/new/choose)
+When the container is up and running, you should be to view the website at [localhost:3000](http://localhost:3000)
+    
+## Missing Features?
 
-### 👷‍♂️ Want to get more hands on?
+Feel free to add issues to our [issue tracker](https://github.com/td-org-uit-no/tdctl-frontend/issues), or create your own  [pull request](#Contributing).
+## Contributing
 
-#### 💅 Formatting the code
+Contributions are always welcome!
 
-We use [Prettier](https://prettier.io/) to format the code in this codebase.
-Prettier is available for most code editors, and also available through the
-command line. Either install it globally on your system, or use the projects
-prettier by running `yarn run prettier --write .` from the root of the repository
-after installing the dependencies with `yarn install`
+See [`contributing.md`](./CONTRIBUTING.md) for ways to get started.
```

Comment:
IMO we should just have the contributing file only in the API. In case of needing to update `CONTRIBUTING.md`, we then need to update both in the api-repo and here, and I feel it's easy to forget to update both

Reply:
However, it could be easier to see the contributing file if it's located here as well, in case someone does not scroll all the way down in the README. I don't know, you can decide

Reply:
Okay, yeah then we'll maintain two separate versions

Reply:
I think the motivation is to avoid redirecting to a new page when you click the link. I agree that the two files can easily diverge so I guess a redir to tdctl_api could be better